docs/THEOplayer/results/issues.md

```diff
@@ -0,0 +1,41 @@
+# Issues
+
+## Issue #1
+
+Live Stream with Low Latency
+
+Feature is expected to be supported at the end of 2020.
```

Comment:
Is there an Issue link on the TheoPlayer site that this can point to?

Reply:
Only internally. However, once supported, we can update our observations here.